radixdlt-java/src/main/java/com/radixdlt/client/core/ledger/AtomStore.java

```diff
@@ -48,4 +43,23 @@
 	 * @return a never ending observable of updates
 	 */
 	Observable<AtomObservation> getAtomObservations(RadixAddress address);
+
+	/**
+	 * Retrieve the current set of validated up particles at a given shardable.
+	 * @param address the address to get the particles under
+	 * @return a stream of all up particles of the current local view
+	 */
+	Stream<Particle> getUpParticles(RadixAddress address, String uuid);
```

Comment:
I think we should add a default variant with uuid as null
Also could add `Nullable` annotation to uuid